benchmarks/python/test_transformer.py

```diff
@@ -430,203 +342,201 @@ def test_transformer_forward(
         create_transformer_forward(fd)
 
     inputs = [
-        2757501781750758,
-        29,
-        2757501781750758,
-        30,
-        2757501781750758,
-        31,
-        torch.randn((36864,), dtype=torch.bfloat16, device="cuda:0").as_strided(
-            (36864,), (1,)
+        torch.randn((25165824,), dtype=torch.bfloat16, device="cuda:0").as_strided(
+            (1, 2048, 12288), (25165824, 12288, 1)
```

Comment:
nit: For all the inputs, can we declare them directly using `torch.randn(size)` instead of the for easier readability.

Reply:
For my own notes: `s/randn((\(\d\+\),)/randn(\1/cg`